README.md

```diff
@@ -40,6 +40,14 @@ npm run apply-entity-changes
 
 Note that if you wish to resolve CNAME's, node version 12+ is required. You can disable CNAME resolution by setting the option treatCnameAsFirstParty=true and keepFirstParty=false in the config file.
 
+### Postgresql data source
+
+Crawler data can also be read from a PostgreSQL database. Simply set the `crawlerDataLoc` to `postgres`, and set the `crawlId` and `region` options. Database details should be provided via environment variables, for example with `envdir`:
```

Comment:
Thanks for documenting this! One nit: I'd drop `simply`, nothing is simple in CS ;)

Reply:
Updated